src/main/webapp/search-interview.js

```diff
@@ -18,42 +18,67 @@ function onSearchInterviewLoad() {
   loginInfo.then(getUserOrRedirectRegistration);
 }
 
-// Should query Datastore for appropriate interviews and render them on the
+// Queries Datastore for available interview times and renders them on the
 // page.
 function loadInterviews() {
-  // TODO: fetch and get all interviews instead of unhiding something hidden
   const searchResultsDiv = document.getElementById("search-results");
   searchResultsDiv.removeAttribute("hidden");
+
+  fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}`)
+  .then(response => response.text())
+    .then(interviewTimes => {
+      interviewTimesDiv().innerHTML = interviewTimes;
+    });
+}
+
+function interviewTimesDiv() {
+  return document.getElementById('interview-times-container');
+}
+
+function browserTimezoneOffset() {
+  let date = new Date();
+  return (-1) * date.getTimezoneOffset();
 }
 
 // Confirms interview selection with user and sends this selection to Datastore 
 // if confirmed.
 function selectInterview(interviewer) {
   if (confirm(
-    // TOOD: fill in these times dynamically from ids in the .jsp file.
-      `You selected: Sunday 7/5 from 6:30 PM - 7:30 PM with a ` +
-      `${interviewer.getAttribute("data-company")} ` +
+      `You selected: ${interviewer.getAttribute('data-date')} from ` +
+      `${interviewer.getAttribute('data-time')} with a ` +
+      `${interviewer.getAttribute('data-company')} ` +
       `${interviewer.getAttribute('data-job')}. ` +
       `Click OK if you wish to proceed.`)) {
     alert(
-      `You have scheduled an interview on Sunday 7/5 from 6:30 PM - 7:30 ` +
-      `PM with a ${interviewer.getAttribute('data-company')} ` +
+      `You have scheduled an interview on ${interviewer.getAttribute('data-date')}` +
+      ` from ${interviewer.getAttribute('data-time')} ` +
+      `with a ${interviewer.getAttribute('data-company')} ` +
       `${interviewer.getAttribute('data-job')}. Check your email for more ` +
       `information.`);
-    // TODO: Call a servlet to save this selection.
-    location.reload();
+    // TODO: Call a servlet to save this selection. Using interviewer.getAttribute('data-utc').
+    // TODO: Change this to data-id once refactor is merged
+    let requestObject = {
+      company: interviewer.getAttribute('data-company'),
+      job: interviewer.getAttribute('data-job'),
+      utc: interviewer.getAttribute('data-utc')
+    };
+    let requestBody = JSON.stringify(requestObject);
+    let request = new Request('/scheduled-interviews', {method: 'POST', body: requestBody});
+    fetch(request).then(unused => {window.location.replace('/scheduled-interviews.html');});
```

Comment:
Q: why `unused`? What's the meaning behind this?

Reply:
Once an interview has been scheduled, redirect the user to the scheduled interviews page.

Reply:
Still confused. What's not used?

Reply:
Whatever the fetch returns (typically the response I think) is unused here because we're not doing anything with info from the servlet apart from redirecting when the fetch is finished.